### ISSUE TITLE
[Feature]: ERC1155Holder

### ISSUE THREAD

**Issue by bidzyyys** [OpenZeppelin/rust-contracts-stylus#363]
### What is the feature you would like to see?

Add ERC1155Holder contract ([solidity](https://docs.openzeppelin.com/contracts/5.x/api/token/erc1155#ERC1155Holder)).

### Contribution Guidelines

- [X] I agree to follow this project's Contribution Guidelines

**Comment by Ifechukwudaniel** [comment]
@bidzyyys  can you assign this to me

**Comment by 0xNeshi** [comment]
@Ifechukwudaniel thanks for your interest!

Since your `ERC4626` PR #465 is still in progress and needs some review items addressed, let's get that one merged first before starting on a new issue. 
Once it's completed, we'll be happy to assign you this one.

**Comment by Ifechukwudaniel** [comment]
Ok , I will do so
